client/src/components/Workflow/Editor/NodeOutput.vue

```diff
@@ -304,7 +320,17 @@ const removeTagsAction = computed(() => {
                     @click="onToggleVisible">
                     <i :class="['mark-terminal', visibleClass]" />
                 </div>
-                {{ label }}
+                <span class="flex-gapx-1">
+                    <span
+                        v-b-tooltip
+                        :title="labelToolTipTitle"
+                        :class="labelClass"
+                        :disabled="!isDuplicateLabel"
+                        style="display: inline-block">
+                        {{ label }}
+                    </span>
+                    <span style="display: inline-block"> ({{ extensions.join(", ") }}) </span>
+                </span>
```

Comment:
- `flex-gapx-1` only works if display is set to flex, so it was not needed
- The inline block display is only required on the first element
- It is preferable to style with classes, rather than inline-styles
- Adding `rounded` to the inline-block brings the style more in line with the rest of the workflow editor

```suggestion
                <span>
                    <span
                        v-b-tooltip
                        :title="labelToolTipTitle"
                        class="d-inline-block rounded"
                        :class="labelClass"
                        :disabled="!isDuplicateLabel">
                        {{ label }}
                    </span>
                    <span> ({{ extensions.join(", ") }}) </span>
                </span>
```

Reply:
Awesome, that's great, learned a bit there!